source/documentation/monitoring_apps/metrics.md

```diff
@@ -2,94 +2,56 @@
-
-	```
-	#!/usr/bin/env bash
-	set -ue
+- CPU
+- RAM
+- disk usage data
+- memory usage data
+- app crashes
+- app requests
+- app response times
 
-	echo "
-	global:
-	  scrape_interval: 1m
-	  evaluation_interval: 1m
-	  scrape_timeout: 1m
+Before you set up the PaaS Prometheus exporter app, you will need:
 
-	scrape_configs:
-	  - job_name: PaaS
-	    bearer_token: $(cf oauth-token | sed 's/bearer //')
-	    scheme: https
-	    static_configs:
-	      - targets:
-	        - API_ENDPOINT:443
-	" > prometheus.yml
+- a monitoring system to store the metrics with an accompanying metrics endpoint
+- a live Cloud Foundry account assigned to the spaces you want to receive metrics on
 
-	docker run --publish 9090:9090 \
-	           --volume "$PWD/prometheus.yml:/etc/prometheus/prometheus.yml" \
-	           prom/prometheus
-	```
-	where API_ENDPOINT is:
-	- `https://metrics.cloud.service.gov.uk/metrics` for the Ireland region
-	- `https://metrics.london.cloud.service.gov.uk/metrics` for the London region
+We recommend that this Cloud Foundry account:
 
-1. Make the script executable:
+- uses the [`SpaceAuditor` role](/orgs_spaces_users.html#space-auditor) as this role has the minimum permissions needed to meet the requirements of the metrics exporter app
+- is separate to your primary Cloud Foundry account
 
-	```
-	chmod +x test-metrics.sh
-	```
+To set up the PaaS Prometheus exporter app:
 
-1. Execute the script:
+1. Clone the [PaaS Prometheus exporter repository](https://github.com/alphagov/paas-prometheus-exporter).
+1. [Push the PaaS Prometheus exporter app](/deploying_apps.html#deploying-public-apps) to Cloud Foundry without starting the app by running `cf push --no-start prometheus-exporter`.
+1. Set the following mandatory environment variables in the metrics exporter app by running `cf set-env prometheus-exporter NAME VALUE`:
 
-	```
-	./test-metrics.sh
-	```
+	|Name|Value|
+	|:---|:---|
+	|`API_ENDPOINT`|- `https://api.cloud.service.gov.uk` for Ireland<br>- `https://api.london.cloud.service.gov.uk` for London|
+	|`USERNAME`|Cloud Foundry User|
+	|`PASSWORD`|Cloud Foundry Password|
 
-	If the script executes successfully, you will see the message:
+	You should use the `cf set-env` command for these mandatory variables as they contain secret information, and this method will keep them secure.
 
-	```
-	msg="Server is ready to receive web requests."
-	```
+	You can also set environment variables by amending the manifest file. We recommend that you use this method for optional environment variables that do not contain secret information. Refer to the [PaaS Prometheus exporter repository](https://github.com/alphagov/paas-prometheus-exporter) for more information.
 
-1. Open your web browser and go to `http://localhost:9090/targets`. You should see the local Prometheus instance running in a Docker container and receiving metrics.
+1. Start the metrics exporter app by running `cf start prometheus-exporter`.
```

Comment:
I think we need to mention:

```
1. Configure the application metrics endpoint in your prometheus scrape configuration
``` 

The metrics endpoint would be the url of the paas-metrics-exporter as deployed before: https://docs.cloud.service.gov.uk/deploying_apps.html#names-routes-and-domains + `/metrics`.

For instance: `https://paas-metrics-exporter-my-org.cloudapps.digital/metrics`